contracts/SmartLoan.sol

```diff
@@ -128,20 +128,43 @@ contract SmartLoan is OwnableUpgradeable, PriceAwareUpgradeable, ReentrancyGuard
   }
 
 
-  /**
-  * This function role is to sell part/all of the available assets in order to bring the loan back to a solvent state.
-  *
-  **/
-  function sellout(uint256 repayAmount) external successfullSellout {
+  function selloutLoan() external onlyOwner {
+    uint256 debt = getDebt();
+    // Extra 10% to sellout all assets despite differences between an oracle's and exchange valuations.
+    uint256 targetPostSelloutAvaxValue = getTotalValue() * 11 / 10;
```

Comment:
I don't like it. Seems like a dirty hack. 
How about implementing an additional fullSellout method that will iterate across all the assets and perform nonSolventAssetSale of full token balance, sth like (please verify syntax):
for (uint i = 0; i < assets.length; i++) {
  nonSolventAssetSale(assets[i], ERC20(assets[I]).balanceOf(this), 0);
}

Reply:
I was thinking about the same. Please check the latest commit where I implemented the updated approach.